### PR TITLE
Models

### PR DESCRIPTION
**Pull request by Scott-Borecki**
### What I Did:
- See PR 25
- Migrate database and create Models
- Add validations

### Road Blocks
- None

### Test Coverage
- [X] I have tested this code with RSpec

### Questions/Notes:
- None

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::API
+  include ExceptionHandler
```
Comment (carolinectan) [inline app/controllers/application_controller.rb:2]: Nice use of modules to dry up code

File: app/controllers/concerns/exception_handler.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# See spec/concerns/response.rb for #json_response helper.
+module ExceptionHandler
+  extend ActiveSupport::Concern
+
+  included do
+    # Raised when Active Record cannot find a record by given id or set of ids.
+    # TODO: Fix errors so it can return multiple errors as elements of an array.
+    rescue_from ActiveRecord::RecordNotFound do |e|
+      json_response(
```
Comment (carolinectan) [inline app/controllers/concerns/exception_handler.rb:9]: Good call on customizing the response message!

**Comment by carolinectan** [comment]
No merge conflicts. Good use of modules.

**Review by tvaroglu — APPROVED** [review]
Fantastic work, again, amazing use of comments to provide clarity for helpers, and extremely thorough, DRY, readable, and tested code.

**Review by carolinectan — APPROVED** [review]
Great work!

**Pull request merged**